rest-service/manager_rest/rest/resources_v3_1/license.py

```diff
@@ -38,6 +45,34 @@ def get(self, license_handler, pagination=None):
         return license_handler.list_license()
 
 
+class LicenseCommunity(SecuredResource):
+    @marshal_with(responses_v3.License)
+    @authorize('license_upload')
+    def post(self):
+        """
+        Store the Customer ID recieved from Hubspot in the Manager.
+        """
+        request_dict = get_json_and_verify_params({
+            'customer_id': {'type': text_type},
+        })
+        sm = get_storage_manager()
+        if len(sm.list(models.License)) > 0:
+            raise ConflictError(
+                'A Customer ID already exists for this manager')
+        return sm.put(models.License(customer_id=request_dict['customer_id']))
```

Comment:
is `customer_id` the only interesting thing here?

Reply:
currently at least